sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -136,6 +139,7 @@
         }
         versionCreated = getVersionCreated(mappingMap);
         versionUpgraded = getVersionUpgraded(mappingMap);
+        closed = getClosed(mappingMap);
```

Comment:
In the partitioned table case I think we create the IndexMetaData manually because it doesn't really exist, so we could set the state accordingly.

I think it would be simpler to use the enum flag instead of accessing the property in the mappingMap